### ISSUE TITLE
Bug in HP_model ECON_IHX variante A

### ISSUE THREAD

**Issue by bilwint** [jfreissmann/heatpumps#12]
The name of the python file is different from the Heat pump class code. So when we do simulation, it picks correct topology image but wrong heat pump class. 
For example, python file name is HeatPumpEconIHX.py, but the Heat pump class code in it is for the HeatPumpIHXEcon.

**Comment by jfreissmann** [comment]
I fixed the issue with the mixup between topology schema and heat pump class of the series economizers. Additionaly, I changed the naming convention of variant "A" and "B" according to how we named these topologies in other work. What needs to be done still is the complete renaming of the parallel compression economizer variants according to this naming convention.

The implementation of variant "B" of the parallel compression econmizer with ihx right after the econ is out of the scope of this issue I think.

**Comment by jfreissmann** [comment]
I finished the enforcement of the ihx naming convention for the parallel compression economizers. For the pc heat pumps with two ihx's, I went with e.g. `ihx_pc_econ_closed_ihx` to align it with the naming of heat pumps with one ihx. If you have any objections to that, I will reopen the issue. Otherwise I think this is done for now.

**Comment by bilwint** [comment]
HeatPumpIHXPC - this naming means, the IHX is immediate after the condenser and not after economizer, right? if so the naming for this model is wrong. Because the connection inside this model in a such a way that the IHX is after economizer.

**Comment by jfreissmann** [comment]
Yeah, you're right. So the real problem with the HeatPumpPCIHX is, that it's topology schemas are wrong...
I'll reopen the issue, undo my changes of renaming it and then we'll fix the schemas.

**Comment by jfreissmann** [comment]
I copied the parallel compression ihx heat pump class and remodelled it to fit its name. Now both variants exist and the old one (now named back to `HeatPumpPCIHX`) only needs the topology schemas. @maltefritz will create them and then the issue should be done.

**Comment by maltefritz** [comment]
I created topology schemas for ``HeatPumpPCIHX``. Now everything should be correct.

Thank you for your work.
